pkg/exporter/metrics.go

```diff
@@ -85,7 +85,8 @@ func (e *Exporter) MetricsHandler(w http.ResponseWriter, r *http.Request) {
 	output, exitCode, err := runScript(scriptName, e.Logger, e.logEnv, timeout, e.Config.GetTimeoutEnforced(scriptName), runArgs, runEnv)
 	if err != nil {
 		successStatus = 0
-		level.Error(e.Logger).Log("msg", "Run script failed", "err", err, "script", scriptName)
+		// useless? runScript already print a more accurate error message
+		// level.Error(e.Logger).Log("msg", "Run script failed", "err", err, "script", scriptName)
```

Comment:
Can you remove these log lines completely please.

Reply:
of course

I will do